chromium_src/chrome/browser/ui/tabs/tab_strip_model_observer.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_TABS_TAB_STRIP_MODEL_OBSERVER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_TABS_TAB_STRIP_MODEL_OBSERVER_H_
+
+class BraveTabStripModel;
+
+// Add virtual method to allow tab strip controllers to react upon MRU cycling
+#define BRAVE_TAB_STRIP_MODEL_OBSERVER_H_ \
+  virtual void StartMRUCycling(BraveTabStripModel* brave_tab_strip_model) {}
```

Comment:
I think TabStripModelObserver is not good place to add this method.
The reason is TabStripModel has multiple TabStripModelObserver.
I think we only need one instance that implement this per TabStripModel.
Instead of expanding `TabStripModelObserver`, how about introducing new interface?

Reply:
I chose to use `TabStripModelObserver` because I thought that whenever `TabStripModel` needs to dispatch an action it does it through an observer. Instead, I could create a `MRUCyclingController` and move all the key handling logic there.

Reply:
I implemented that, please let me know what you think.